open-metadata-implementation/access-services/cognos/cognos-api/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<artifactId>cognos</artifactId>
+		<groupId>org.odpi.egeria</groupId>
+		<version>1.8-SNAPSHOT</version>
+	</parent>
+
+	<name>Cognos OMAS API</name>
+	<description>
+        Shared beans for the Cognos Open Metadata Access Service (OMAS).
+	</description>
+
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>cognos-api</artifactId>
+
```

Comment:
Can you add an scm fragment
```
   <scm>
         <connection>scm:git:git://github.com/odpi/egeria.git</connection>
         <developerConnection>scm:git:ssh://github.com/odpi/egeria.git</developerConnection>
         <url>http://github.com/odpi/egeria/tree/master</url>
     </scm>
```
We have in our other poms, and in some cases is needed to keep our jfrog publishing process happy

Reply:
done https://github.com/odpi/egeria/pull/3044/commits/2c7f52af9a67feb496eedd853b634a9b5f478b2f

Reply:
thanks